maploom/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = (1, 5, 60)
```

Comment:
This is more of a question for Ami.

If you're trying to get the version for MapLoom (and not just this package) I'd suggest using the package.json which is automatically updated by Jenkins.   The versions between this and package.json should match.  (*should*)